serverless/serverless-support.adoc

```diff
@@ -0,0 +1,24 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-support"]
+= {ServerlessProductName} support
+:context: serverless-support
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+include::modules/serverless-getting-support.adoc[leveloffset=+1]
+
+== Collecting {ServerlessProductName} data for Red Hat Support
+
+When opening a support case, it is helpful to provide debugging
+information about your cluster to Red Hat Support.
+
+The `must-gather` tool enables you to collect diagnostic information about your
+{product-title} cluster, including virtual machines and other data related to
```

Comment:
```suggestion
{product-title} cluster, including data related to
```

Reply:
I'm not aware of virtual machines related to Serverless. This is probably specific to Service Mesh.